e2e/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "../out-tsc/e2e",
     "module": "commonjs",
-    "target": "es5",
+    "target": "es2018",
```

Comment:
In the tsconfig.json the value is `es2020`. Is there an equivalent here or this is the latest and matching?

Reply:
That is a good point. This was automatically updated by angular CLI so I assume it is correct. However, currently there's no e2e tests so I don't think we have to worry right now

Reply:
OK, I already approved it, just thought I would ask it.